ocs_ci/ocs/cluster.py

```diff
@@ -3602,3 +3602,21 @@ def bring_down_mds_memory_usage_gradually():
     assert (
         time_elapsed <= 1800
     ), "Memory usage remained high for more than 30 minutes. Failed to bring down the memory usage of MDS"
+
+
+def get_used_and_total_capacity_in_gibibytes():
```

Comment:
we already have get_ceph_capacity and get_ceph_free_capacity. Please check if they can be re-used

Reply:
I cannot use these functions due to below concerns.
1. Existing functions calculated by removing the replicas, I want the capacity without removing replicas to calculate accurate values in estimated days.
get_ceph_capacity  = 799.984375
get_ceph_free_capacity  = 761.4427757263184 --> I want used capacity where I need to do total minus free again in a function and multiply with replica

**get_used_and_total_capacity_in_gibibytes = (115.62479782104492, 2399.953125)**

2. I run these functions almost in all tests. I jsut want to avoid initializing  CephCluster class in test case just for these functions though these two are directly not useful for me.